lib/ansible/modules/cloud/ovirt/ovirt_clusters.py

```diff
@@ -208,6 +208,9 @@
         description:
             - "The compatibility version of the cluster. All hosts in this
                cluster must support at least this compatibility version."
+    mac_pool:
+        description:
+            - "MAC pool to be used by this cluster."
```

Comment:
Add a note that it's supported since 4.1 please.

If user will use unsupported Python SDK version he should get attribute error, and if user will use unsupported oVirt version he should get fail response